pkg/vpc/vpc_test.go

```diff
@@ -258,9 +258,12 @@ var _ = Describe("VPC - Import VPC", func() {
 				return input != nil
 			})).Return(mockResultFn, vpcCase.describeVPCError)
 
-			if err := importVPC(p, vpcCase.cfg, vpcCase.id); err != nil {
+			err := importVPC(p, vpcCase.cfg, vpcCase.id)
+			if vpcCase.error != nil {
+				Expect(err).To(HaveOccurred())
```

Comment:
this check is redundant but nbd 🤷‍♀️

Reply:
In the `if` statement I'm checking what's expected while the `Expect` asserts it to be the case

Reply:
On the next line you are checking the error message, so the has occurred bit is sort-of implied right?

Reply:
but really doesn;t matter 🙃

Reply:
`.Error()` was being called so I need to check it's not null but apparently `MatchError` accepts strings so I've used that instead and also fixed the existing logic error everywhere